sqlalchemy_history/operation.py

```diff
@@ -72,7 +71,7 @@ def entities(self):
         return set(key[0] for key, _ in self.iteritems())
 
     def iteritems(self):
-        return six.iteritems(self.objects)
+        return iter(self.objects)
```

Comment:
`six.iteritems` gives is a iterator of `key, val` pairs.
But `iter()` will just give us an iterator for keys

So, this is not equivalent
In fact - I think we can just remove this function because `iteritems` is not a function people in py3 use after `dict_items` was added in py3